index.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
       'silvermine/no-multiline-var-declarations': 'error',
       'silvermine/indent': [ 'error', 3, { 'VariableDeclaratorOffset': { 'var': 1, 'let': 1, 'const': 3 }, 'SwitchCase': 1 } ],
 
-      'comma-dangle': [ 'error', 'only-multiline' ],
+      'comma-dangle': [ 'error', 'always-multiline' ],
```

Comment:
I think a better approach might be:

 1) leave the rule here in index.js as `only-multiline`
 2) in our node.js rules config, override it to be `always-multiline`.

My thinking is that we actually have always gone the other way in browser JS: enforce no trailing comma since some browsers puke if there is one.

@yokuze do you have thoughts on this?